abdiff/core/init_run.py

```diff
@@ -0,0 +1,31 @@
+"""abdiff.core.init_run"""
+
+import datetime
+import logging
+import os
+from pathlib import Path
+
+from abdiff.config import Config
+from abdiff.core.utils import read_job_json, update_or_create_run_json
+
+CONFIG = Config()
+
+logger = logging.getLogger(__name__)
+
+
+def init_run(
+    job_directory: str,
+    message: str | None = None,
+) -> str:
+    """Function to initialize a new Run as part of a parent Job."""
+    run_timestamp = datetime.datetime.now(tz=datetime.UTC).strftime("%Y-%m-%d_%H-%M-%S")
+    run_directory = str(Path(job_directory) / "runs" / run_timestamp)
+    os.makedirs(run_directory)
+    logger.info(f"Run working directory created: {run_directory}")
```

Comment:
Optional: I don't think much would be lost by dropping `working` from the log message and we gain symmetry with the variable name but completely your call

Reply:
Agreed, changed.